R/simulation-run-options.R

```diff
@@ -35,9 +41,16 @@ SimulationRunOptions <- R6::R6Class(
       ospsuite.utils::ospPrintClass(self)
       ospsuite.utils::ospPrintItems(list(
         "numberOfCores" = self$numberOfCores,
-        "checkForNegativeValues" = self$checkForNegativeValues,
         "showProgress" = self$showProgress
       ))
+    },
+    #' @description
+    #' Get the checkForNegativeValues property (deprecated)
+    #' @return The value of checkForNegativeValues from the solver
+    .getCheckForNegativeValues = function() {
```

Comment:
How does this insure backwards compatibility? The field `checkForNegativeValues` is gone, so old code using it will still break. Or what am I missing?